notebooks/Primer/Primer.ipynb

```diff
@@ -2,10 +2,7 @@
  "cells": [
```

Comment:
That's what I use to denote we should move over to exercise #2 in the exercise notebook, which is where the instructions live. Thoughts?